plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java

```diff
@@ -43,23 +46,21 @@ public void setUp() {
     doc = pluginXmlDocument.get();
   }
 
-  protected Document getDocument() {
+  protected final Document getDocument() {
     return doc;
-  }
-  
-  // Generic tests that should be true of all plugin.xml files
+  } 
   
   @Test
-  public void testRootElementIsPlugin() {
+  public final void testRootElementIsPlugin() {
```

Comment:
Hmm... These are just tests. Is adding `final` to unit tests our new convention?